src/features/classification/AddClassificationBadge.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/variables';
+@import './mixins';
+
+.bdl-AddClassificationBadge {
+    @include bdl-ClassificationBadge;
+
+    background-color: $bdl-gray-02;
+    border-color: $bdl-gray-20;
+
+    &:hover {
```

Comment:
Is `&:focus` needed to support keyboard navigation, as well?

Reply:
It does get the blue outline because its wrapped within a button with a on click. So the badge itself is fine. We can tho choose to move the button in here and pass in click handlers/resin tags